test/functional/element_creation.js

```diff
@@ -61,6 +63,36 @@ describe('element creation', () => {
       expect(el.getAttribute('data-foo')).to.equal('Hello');
       expect(el.getAttribute('data-bar')).to.equal('World');
     });
+
+    it('should return created node', () => {
```

Comment:
Any advice on the test description: "patching an element with an existing document tree [should return DOM node] [from elementOpen]"?